jsx/PaginationLinks.js

```diff
@@ -0,0 +1,69 @@
+//var PureRenderMixin = require('react/addons').addons.PureRenderMixin;
+
+PaginationLinks = React.createClass({
+    mixins: [React.addons.PureRenderMixin],
+    propTypes: {
+        onChangePage: React.PropTypes.func,
+        Total: React.PropTypes.number.isRequired
+    },
+    getDefaultProps: function() {
+        return {
+            'RowsPerPage' : 10,
```

Comment:
Nvm I see that you can pass a value in for this. Should probably pass the value from the config table though, non?